HeliPort/CredentialsManager.swift

```diff
@@ -32,6 +33,17 @@ final class CredentialsManager {
         Log.debug("Loading password for network \(network.ssid)")
         return password
     }
+
+    func getSavedNetworks() -> [NetworkInfo] {
+        var list: [NetworkInfo] = []
+        for ssid in keychain.allKeys() {
+            // TODO: unserialize networkInfo
+            let network = NetworkInfo(ssid: ssid, connected: false, rssi: 0)
+            network.auth.security = NetworkInfo.AuthSecurity.CCMP.rawValue
+            list.append(network)
+        }
+        return list
```

Comment:
```suggestion
        return keychain.allKeys().map { ssid in
            // TODO: unserialize networkInfo
            let network = NetworkInfo(ssid: ssid, connected: false, rssi: 0)
            network.auth.security = NetworkInfo.AuthSecurity.CCMP.rawValue
            return network
        }
```

bit more functional to keep the "theme" in the network manager class